WordPress/Classes/Models/Blog.m

```diff
@@ -691,7 +691,8 @@ - (BOOL)supportsPluginManagement
 - (BOOL)supportsStories
```

Comment:
I originally considered checking the `isPad` and feature removal conditions in a specific helper within the [`StoryAction` class](https://github.com/wordpress-mobile/WordPress-iOS/blob/3eb3666f8cf358292aac1ace26b58dd694294d04/WordPress/Classes/ViewRelated/System/Floating%20Create%20Button/SheetActions.swift#L41). But I finally ended up using the `supports` functionality of the Blog model as the logic seemed closer to me. In any case, please let me know if there's a better approach for this, thanks!